lib/jumubase_web/views/encoders/ecto_encoders.ex

```diff
@@ -1,18 +1,29 @@
-# The following Poison.Encoder implementations ensure that JSON-encoding an
+# The following Jason.Encoder implementations ensure that JSON-encoding an
 # Ecto changeset or record doesn't result in errors because of un-encodable
 # constructs (such as keyword lists).
 
-defimpl Poison.Encoder, for: Ecto.Changeset do
+require Protocol
+alias Jumubase.Showtime.{Appearance, Participant, Performance, Piece}
+
+# White-list schema fields that may be exposed via JSON
```

Comment:
My idea was to list the "allowed" fields here instead of spreading that logic around the schemas with `@derive`. But perhaps this isn't the way to go either, and we should filter fields in the still-unwritten data mapping function?